library/libraries.js

```diff
@@ -993,8 +993,8 @@ var d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests = {
         icon: 'vue',
         url: 'http://vuejs.org/',
         test: function(win) {
-            if (win.Vue && win.Vue.compiler) {
-                return { version: 'N/A' };
+            if (win.Vue && win.Vue.version) {
```

Comment:
This key is part of the API, but is not documented, I have sent a PR to include in the documentation.